index.html

```diff
@@ -0,0 +1,720 @@
+            </article>
+
+            <article class="article-round">
+                <img class="article-round-img"
+                    src="https://index.tnwcdn.com/images/6a652bbb8a1a8ee7ac6ed633991a9b1105333b51.png" alt="">
+                <h2 class="header">SubjectWell</h2>
+                <section class="article-round-head">
+                    <h2 class="heading-none">Header</h2>
+                    <span>$ 10M<br></span>
+                    <small>SERIES A</small>>
+
+                </section>
+                <a href="#">
+                    <i class="fas fa-tag"></i>
+                    <span>Security</span>
+                </a>
+                <a href="#">
+                    <i class="fas fa-briefcase"></i>
+                    <span>August Capital, +2</span>
+                </a>
+                <p>SubjectWell nabs $10 mln Series A <span style="color:gray">on PE Hub </span></p>
+            </article>
+
+
+            <article class="article-round">
+                <img class="article-round-img"
+                    src="https://index.tnwcdn.com/images/560289d4be5a1539c9e6c8f2bba15eb80a61ebed.jpg" alt="">
+                <h2 class="header">SubjectWell</h2>
+                <section class="article-round-head">
+                    <h2 class="heading-none">Header</h2>
+                    <span>$ 10M<br></span>
+                    <small>SERIES A</small>
+                </section>
+                <a href="#">
+                    <i class="fas fa-tag"></i>
+                    <span>Security</span>
+                </a>
+                <a href="#">
+                    <i class="fas fa-briefcase"></i>
+                    <span>August Capital, +2</span>
+                </a>
+                <p>SubjectWell nabs $10 mln Series A <span style="color:gray">on PE Hub </span></p>
+            </article>
+
+            <article class="article-round">
+                <img class="article-round-img"
+                    src="https://index.tnwcdn.com/images/be17d6d4a637e935624f0501647b27e8807d93db.jpg" alt="">
+                <h2 class="header">SubjectWell</h2>
+                <section class="article-round-head">
+                    <h2 class="heading-none">Header</h2>
+                    <span>$ 10M<br></span>
+                    <small>SERIES A</small>
+                </section>
+                <a href="#">
+                    <i class="fas fa-tag"></i>
+                    <span>Security</span>
+                </a>
+                <a href="#">
+                    <i class="fas fa-briefcase"></i>
+                    <span>August Capital, +2</span>
+                </a>
+                <p>SubjectWell nabs $10 mln Series A <span style="color:gray">on PE Hub </span></p>
+            </article>
+        </section>
+
+        <section class="latest-info">
+            <h2 class="heading-none">Header</h2>
+            <article>
```

Comment:
Maybe these can be wrapped `section` tags since technically they are sections in there own. containing there own articles.

Reply:
Do this for all within this section